### ISSUE TITLE
Support of retina display in MacOS

### ISSUE THREAD

**Issue by k-lepa** [Dwarf-Therapist/Dwarf-Therapist#44]
Hi,

Add support of retina for MacOS build in Info.plist, pls:
```xml
<key>NSPrincipalClass</key>
<string>NSApplication</string>
<key>NSHighResolutionCapable</key>
<string>True</string>
```

Thanks

**Comment by cvuchener** [comment]
I don't know what to do with that. If you do, please write a pull request updating the macOS scripts.

Also, I think Qt only supports high dpi since 5.6, the content of info.plist may need to be configured depending on the Qt version used.

**Comment by cvuchener** [comment]
fixed in #45
